plugin/helpers.go

```diff
@@ -79,6 +79,19 @@ type Helpers interface {
 	// Minimum server version: 5.2
 	ShouldProcessMessage(post *model.Post, options ...ShouldProcessMessageOption) (bool, error)
```

Comment:
`ShouldProcessMessage` uses a function options. Hence, we can add a new option without changing the signature. What do you think about a `BotID` option?

Reply:
As I was mentioning in the previous comment - the options set is not general, it is called `ShouldProcessMessageOption` (and the current list of options: `AllowWebhook`, etc, support this naming), so BotID doesn't really fit (to me).

Reply:
Could you please elaborate why `BotID` isn't a fitting option to you? The whole idea of functional options is to support addtional options when they are needed without changing the signature.

Reply:
@hqnzei - I guess I see BotID (and other `id`s) as essentially function params versus options (like a "should I do this or not" option). Wasn't aware that the primary intent of the Options set is solve the signature issue, don't really agree with that but I'll align to it since that is the existing pattern.